patches/server/0880-Allow-usage-of-MiniMessage-in-server-configuration.patch

```diff
@@ -0,0 +1,410 @@
+From: Peter Crawley <peterzcrawley@gmail.com>
+Date: Sat, 5 Mar 2022 19:08:14 +0000
+Subject: [PATCH] Allow usage of MiniMessage in server configuration files
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperCommand.java b/src/main/java/com/destroystokyo/paper/PaperCommand.java
+index 76d27205b2bd08ba2b3530b46f0d9b4e2315d82e..73f54651f2bba1ae45e57e9c1fd77f3bd10aeeb5 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperCommand.java
++++ b/src/main/java/com/destroystokyo/paper/PaperCommand.java
+@@ -81,7 +81,7 @@ public class PaperCommand extends Command {
+ 
+     private static boolean testPermission(CommandSender commandSender, String permission) {
+         if (commandSender.hasPermission(BASE_PERM + permission) || commandSender.hasPermission("bukkit.command.paper")) return true;
+-        commandSender.sendMessage(Bukkit.getPermissionMessage()); // Sorry, kashike
++        commandSender.sendMessage(Bukkit.permissionMessage()); // Sorry, kashike
+         return false;
+     }
+ 
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index 2becc9a274eddb1a01655f24f54d5cf72470580d..c9d472222d9f3ac258b22172e989d3998182ae1f 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -1,7 +1,6 @@
+ package com.destroystokyo.paper;
+ 
+ import com.destroystokyo.paper.io.chunk.ChunkTaskManager;
+-import com.google.common.base.Strings;
+ import com.google.common.base.Throwables;
+ 
+ import java.io.File;
+@@ -18,18 +17,23 @@ import java.util.logging.Level;
+ import java.util.regex.Pattern;
+ 
+ import com.google.common.collect.Lists;
++import net.kyori.adventure.text.Component;
++import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+ import net.minecraft.server.MinecraftServer;
+ import org.bukkit.Bukkit;
+-import org.bukkit.ChatColor;
+ import org.bukkit.command.Command;
+ import org.bukkit.configuration.ConfigurationSection;
+ import org.bukkit.configuration.InvalidConfigurationException;
+ import org.bukkit.configuration.file.YamlConfiguration;
+ import co.aikar.timings.Timings;
+ import co.aikar.timings.TimingsManager;
++import org.jetbrains.annotations.Nullable;
+ import org.spigotmc.SpigotConfig;
+ import org.spigotmc.WatchdogThread;
+ 
++import static net.kyori.adventure.text.minimessage.MiniMessage.miniMessage;
++import static net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer.*;
++
+ public class PaperConfig {
+ 
+     private static File CONFIG_FILE;
+@@ -71,8 +75,8 @@ public class PaperConfig {
+         commands.put("paper", new PaperCommand("paper"));
+         commands.put("mspt", new MSPTCommand("mspt"));
+ 
+-        version = getInt("config-version", 25);
+-        set("config-version", 25);
++        version = getInt("config-version", 26);
++        set("config-version", 26);
```

Comment:
This change needs to be done in the very first config patch

Reply:
Moved.